### ISSUE TITLE
If historical data available, current day shows that

### ISSUE THREAD

**Issue by vakesz** [ZsBT/hass-w1000-portal#21]
First try the provided script worked flawlessly but after a few hours the daily stats show the all time import, export values, and only previous days work correctly.

Daily:
![image](https://github.com/ZsBT/hass-w1000-portal/assets/3627611/2d88384f-b2c4-44f2-961a-24e18adb505d)

Previous day:
![image](https://github.com/ZsBT/hass-w1000-portal/assets/3627611/f0ee3494-93a8-45bd-bd02-7ae685917758)

Also since I can't open a PR here, I'd recommend a few changes to make it clearer how to install W1000-portal.
[README.md](https://github.com/ZsBT/hass-w1000-portal/files/11581672/README.md)

**Comment by ZsBT** [comment]
Indeed, the current version keeps the most recent day messy (it is already
reported). My plan is to completely rewrite the algorithm but that requires
all the users to recreate the curves on the EON portal as well.

I hope I'll have the time to do this soon.
